api/src/test/java/org/openmrs/api/ProviderServiceTest.java

```diff
@@ -492,9 +491,8 @@ public void getCountOfProviders_shouldExcludeRetiredProviders() {
 	 * @see ProviderService#getCountOfProviders(String,null)
 	 */
 	@Test
-	@Ignore
 	public void getCountOfProviders_shouldIncludeRetiredProvidersIfIncludeRetiredIsSetToTrue() {
-		assertEquals(4, service.getCountOfProviders("provider").intValue());
+		assertEquals(4, service.getCountOfProviders("provider",true).intValue());
```

Comment:
This is quite nit picky but could you leave a space after the comma and before `true`?

See **8.2 Blank Spaces** [here](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html) Oracle's 'Code Conventions for the Java Programming Language':
>A blank space should appear after commas in argument lists.

Reply:
@mks-d  done as expected :)